setup.py

```diff
@@ -759,21 +759,21 @@ def printException_loggerwarn():
 "Press any key when you are done and ready.", sep="\n")
 input()
 print("Please input your API key below.")
-apikey_input = input("Input here: ")
+apikey_input = str(input("Input here: ")).replace(" ","").replace("\n","").replace("\t","")
 logger.debug("apikey_input: %s" % apikey_input)
-print("", "Just to confirm, the API key you gave me was: " + apikey_input
-      + ".", sep="\n")
+print("", "Just to confirm, the API key you gave me was: '" + apikey_input
+      + "'", sep="\n")

```

Comment:
Is there any reason to add `'` on top of `"`.

The single quotation mark shouldn't replace the period in some places as it's part of the setup dialogue.